MPF.Frontend/ViewModels/MainViewModel.cs

```diff
@@ -2211,29 +2211,61 @@ private bool ValidateBeforeDumping()
             var outputDirectory = Path.GetDirectoryName(_environment!.OutputPath);
             string outputFilename = Path.GetFileName(_environment.OutputPath);
 
-            // If a complete dump already exists
-            bool foundFiles = _environment.FoundAllFiles(outputDirectory, outputFilename);
-            if (foundFiles && _displayUserMessage != null)
+            // If a complete or partial dump already exists
+            bool foundAllFiles = _environment.FoundAllFiles(outputDirectory, outputFilename);
+            if (foundAllFiles && _displayUserMessage != null)
             {
                 bool? mbresult = _displayUserMessage("Overwrite?", "A complete dump already exists! Are you sure you want to overwrite?", 2, true);
                 if (mbresult != true)
                 {
                     LogLn("Dumping aborted!");
                     return false;
                 }
+                // else continue with dump, don't check further
             }
             else
```

Comment:
Can you find a way of avoiding having 3 levels of nesting here? Reading this code is fairly awkward.

Reply:
I can't find a way that doesn't add unnecessary complexity, no